sources/packages/backend/libs/integrations/src/services/sfas/sfas-restriction.service.ts

```diff
@@ -108,6 +124,58 @@ export class SFASRestrictionService
     await this.repo.save(restriction, { reload: false, transaction: false });
   }
 
+  /**
+   * Updates processed to false for the SFAS restrictions corresponding to the provided SFAS Individual Id.
+   * @param individualId provided SFAS Individual Id.
+   */
+  async updateProcessedStatus(individualId: number): Promise<void> {
+    await this.repo.update({ individualId }, { processed: false });
+  }
+
+  /**
+   * Create notifications for newly created legacy student restrictions for the ministry.
+   * @param studentIds student ids for which the legacy restriction notification needs to be sent.
+   * @param entityManager entity manager to execute in transaction.
+   */
+  private async createLegacyRestrictionNotifications(
+    studentIds: number[],
+    entityManager?: EntityManager,
```

Comment:
If an entity manager is a provider it should be used.
If there is no reason for this `entityManager` to be optional it can be changed and used to query the student data.